dynamiqs/smesolve/smesolve.py

```diff
@@ -217,18 +217,18 @@ def smesolve(
         )
```

Comment:
Can you remove the inline comments indicating returned tensors size just below the docstring? Also in sesolve, they're outdated.